app/views/fact_values/index.html.erb

```diff
@@ -7,7 +7,7 @@ end
 %>
 <% title_actions documentation_button('3.5.5FactsandtheENC') %>
 
-<table class="<%= table_css_classes('ellipsis')%>">
+<table class="<%= table_css_classes('table-fixed') %>">
```

Comment:
Do we actually need popovers anywhere here? if so, give the `<th>`s some width so they will not be evenly distributed and give the `ellipsis` class to any `<td>`s that need tooltips.